test/jest/__tests__/investment/investment.spec.ts

```diff
@@ -0,0 +1,77 @@
+import {InvestmentInputFields} from 'components/investment/interfaces/InvestmentInputFields'
+import {describe, expect, it} from '@jest/globals'
+import investment from 'components/investment/investment'
+
+const defaultInput:InvestmentInputFields = {
+  amount:10000,
+  monthCount:12,
+  rate:0.1,
+}
+
+describe('investment',()=>{
```

Comment:
Generally, it should also check edge cases when one of inputs is equal to 0. I know UI doesn't allow to set 0 but calculations should be correct. The UI might be changed in the future